### DISCUSSION TITLE
How do I make it so that you can create multiple towns?

### DISCUSSION THREAD

**Discussion by PixelPingu** [TownyAdvanced/Towny#5192] [Q&A] [answered]
I've been trying to do this for hours now, what should I put in the permissions file?

**Comment by LlmDl** [comment, marked as answer]
A normal player cannot make multiple towns. An admin can do the following:

https://github.com/TownyAdvanced/Towny/wiki/How-Towny-Works#mayors

The short version;
When you have your town set up do `/ta set mayor {townname} NPC`.

Then `/t leave`, then create your second town.

Then you can use `/ta set mayor {townname} npc` & `/t leave` whenever you want to switch towns and then `/ta town {townname} add {yourname}` to add yourself to the previous town.

**Reply by PixelPingu** [reply]
thank you! I'll try this out
